src/components/landing/Form.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import H3 from '@material-tailwind/react/Heading3';
+import Paragraph from '@material-tailwind/react/Paragraph';
+import Input from '@material-tailwind/react/Input';
+import Button from '@material-tailwind/react/Button';
+
+export default function Form() {
+    return (
+        <div className="flex flex-wrap justify-center mt-24">
+            <div className="w-full lg:w-8/12 px-4">
+                <div className="relative flex flex-col min-w-0 break-words w-full mb-6">
+                    <div className="flex-auto p-5 lg:p-10">
+                        <div className="w-full text-center">
+                            <H3 color="gray">Want to work with us?</H3>
+                            <Paragraph color="blueGray">
+                                Complete this form and we will get back to you
+                                in 24 hours.
+                            </Paragraph>
+                        </div>
+                        <form onSubmit={(e) => e.preventDefault()}>
+                            <div className="flex gap-8 mt-16 mb-10">
+                                <Input
+                                    type="text"
+                                    placeholder="Full Name"
+                                    color="lightBlue"
+                                />
+                                <Input
+                                    type="email"
+                                    placeholder="Email Address"
+                                    color="lightBlue"
+                                />
+                            </div>
+
+                            <div
+                                className="relative w-full"
+                                style={{ height: '150px' }}
```

Comment:
Once again, this `style` prop should be changed with one of the Tailwind CSS classes. You can choose between `h-36` and `h-40`, and if they are not ok, please create your own `h-*` class and use that one. For example `h-38` which would add a height of `9.5rem`, or `h-37` which would add a height of `9.25rem`.